Hi @matiasinsaurralde,

There's a failing static analysis check, just because `max` is the name of a builtin function in go now, and there's a local variable shadowing it:

```
task: [lint] .tool/golangci-lint run --tests=false
Error: grype/presenter/models/sort.go:267:2: redefines-builtin-id: redefinition of the built-in function max (revive)
	max := es[0].Percentile
	^
Error: grype/presenter/models/sort.go:270:4: redefines-builtin-id: redefinition of the built-in function max (revive)
			max = e.Percentile
			^
2 issues:
* revive: 2
```

--- a/grype/presenter/models/sort.go
+++ b/grype/presenter/models/sort.go
@@ -264,13 +264,13 @@ func epssPercentile(es []EPSS) float64 {
 	if len(es) == 0 {
 		return 0.0
 	}
-	max := es[0].Percentile
+	maxPercentile := es[0].Percentile
 	for _, e := range es[1:] {
-		if e.Percentile > max {
-			max = e.Percentile
+		if e.Percentile > maxPercentile {
+			maxPercentile = e.Percentile
 		}
 	}
-	return max
+	return maxPercentile
 }
 
 // severityPriority maps severity strings to numeric priority for comparison (the lowest value is most severe)

